### ISSUE TITLE
Issue with the Teacher forcing ratio

### ISSUE THREAD

**Issue by LoKerpiqo** [vincent-leguen/PhyDNet#26]
Hi author, could you please disclose the setting of the teacher forcing ratio used for the human3.6 dataset? and Other training parameters (training epoch, batch_size) if you would like to make them public. 

I tried to reproduce your method on this dataset but got terrible results on testing data. However, the performance on the validation set was good. I am guessing if that's the problem with the teacher forcing ratio that makes inconsistency between training and testing?

Did you observe the performance gap (visually) between the validation and testing set? 

Thank you.
